mkdocs.yml

```diff
@@ -5,7 +5,11 @@ repo_name: PyStemmusScope
 nav:
   - Getting started: index.md
   - Installation: installation_instructions.md
-  - Notebook page: notebooks/run_model_in_notebook.ipynb
+  - Run the model:
+      - Run the model with different options: notebooks/run_model_with_different_options.ipynb
+      - Run the model with different dataset using executable file: notebooks/run_model_with_different_dataset.ipynb
+      - Run the model with different dataset on CRIB: notebooks/run_model_with_different_dataset_on_CRIB.ipynb
```

Comment:
```suggestion
  - Run the model:
      - Different ways to run the model: notebooks/run_model_with_different_options.ipynb
      - Site and global datasets with the executable file: notebooks/run_model_with_different_dataset.ipynb
      - Site and global datasets on CRIB: notebooks/run_model_with_different_dataset_on_CRIB.ipynb
```

Reply:
Do note that these names will not be shown, as they're overridden by the H1 header of the notebooks, so those would have to be modified as well.

So perhaps it's better to not use the full names here, only on the notebooks.